lua/astrocommunity/completion/tabby-nvim/init.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  "TabbyML/vim-tabby",
+  init = function()
+    vim.g.tabby_keybinding_accept = "<C-e>"
+    vim.g.tabby_keybinding_trigger_or_dismiss = "<C-\\>"
+    vim.g.tabby_node_binary = vim.fn.exepath("node")
+  end,
```

Comment:
```suggestion
  dependencies = {
    "AstroNvim/astrocore",
    ---@type AstroCoreOpts
    opts = {
      options = {
        g = {
          tabby_keybinding_accept = "<C-e>",
          tabby_keybinding_trigger_or_dismiss = "<C-\\>",
          tabby_node_binary = vim.fn.exepath "node",
        },
      },
    },
  },
```

Use AstroCore for configuring global options to expose them to the user to modify.

Reply:
@Uzaaft this handles moving the mappings to AstroCore